documentation/en/draft_modulators.md

```diff
@@ -0,0 +1,370 @@
+
+#### 2.1.2 Electromagnetic waves
+
+The electromagnetic waves propagate through the space vacuum withot a material medium. This is due by the oscilations of the electric field in relation with the magnetic field. The electromagnetics waves travel in an approximate speed of 300,000 Km/s, depending on the speed it can be grouped in frequency. This ordering is known as electromagnetic spectrum, it measures the waves frequency. The X-rays, visible light or ultraviolet rays belong in this type.
+  
+#### 2.1.3 Gravitational wave 
+The gravitational waves are disturbances altering the geometry of spacetime, it's commonly represented traveling through space although technically we can say that they're moving through space but that they are disturbances of the spacetime itself.
+
+###  2.2 Radiation
+***********
+#### 2.2.1 Unidimensional wave 
+These waves propagate through space in one direction, as the waves in docks or in strings. If a wave propagtes in only a direction, its wavefronts are plane and parallel.
+    
+#### 2.2.2 Two-dimensional or surface wave 
+The two-dimensional wave propagates in two directions. They can travel in any surface, this is why they can also be called surface wave, like the waves produced in a liquid surface at rest when a rock is let drop in it.
+
+
+#### 2.2.3 Three-dimensional o esféricas
+Son ondas que se propagan en tres direcciones. Las ondas tridimensionales se conocen también como ondas esféricas, porque sus frentes de ondas son esferas concéntricas que salen de la fuente de perturbación expandiéndose en todas direcciones. El sonido es una onda tridimensional. Son ondas tridimensionales las ondas sonoras (mecánicas) y las ondas electromagnéticas.
+**********
+
+### 2.3 Particles movement
+
+#### 2.3.1  Longitudinal waves
+In this type of wave the particles vibrate parallel to the propagation of the wave. For example, the seismic waves, sound waves and compressiong an elastic spring results in a longitudinal wave. 
+
+#### 2.3.2 Transverse waves
+In the transverse waves the particles vibrate perpendicularly to the propagation of the wave. Like the sea waves, waves in a string, and seismic waves.
+
+### 2.4 Periodicity
+
+#### 2.4.1 Periodic wave
+The local disturbance that originates this type of waves it's produced in repetitive cicles, e.g. a sine wave.
+
+#### 2.4.2 Aperiodic Non-periodic wave 
+
+The disturbance that originates them are isolated, or in case it repeats, the following disturbances have different characteristics. These isolated waves are known as pulse wave.
+
+
+# Chapter 2 Modulators and demodulators
+
+Information signals are rarely in an appropiate state for transmission, so that signals must be carry between the transmisor and the receiver on some medium for transmission.
```

Comment:
```suggestion
Information signals are rarely in an appropriate state for transmission, so that signals must be carry between the transmisor and the receiver on some medium for transmission.
```